BLSQ/iaso
# POLIO-939: campaign history show old keys for old campaigns

There were missing info in campaign history because old keys

Related JIRA tickets : POLIO-939

## Self proofreading checklist


## Changes

- added old keys to config and update translations

## How to test

- Go to campaigns
- Edit an old campaign
- See if the keys and values for older fields are stilled displayed in the campaign history

## Print screen / video

https://github.com/BLSQ/iaso/assets/25134301/7a317ecb-1db4-4d06-89b8-119dd698bf89

## Notes
The list of fields was based on this file : [Fields on Campaign](https://docs.google.com/spreadsheets/d/1g1g8i5Om82ixSwOuL18kIpxH2KVWyJcC-3dDh3p_dM0/edit#gid=0)
